docs/linting/TROUBLESHOOTING.md

```diff
@@ -28,12 +28,48 @@ If you don't find an existing extension rule, or the extension rule doesn't work
 > We release a new version our tooling every week.
 > _Please_ ensure that you [check our the latest list of "extension" rules](https://typescript-eslint.io/rules/#extension-rules) **_before_** filing an issue.
 
-## I get errors telling me "The file must be included in at least one of the projects provided"
+## I get errors telling me "ESLint was configured to run ... However, that TSConfig does not / none of those TSConfigs include this file"
+
+### Fixing the Error
```

Comment:
I split this section into these two h3s. Re-reading the un-split explanation felt like a tl;dr.